src/backend/distributed/planner/intermediate_result_pruning.c

```diff
@@ -106,7 +106,7 @@ RecordSubplanExecutionsOnNodes(HTAB *intermediateResultsHash,
 	List *usedSubPlanNodeList = distributedPlan->usedSubPlanNodeList;
 	List *subPlanList = distributedPlan->subPlanList;
 	ListCell *subPlanCell = NULL;
-	int workerNodeCount = GetWorkerNodeCount();
+	int workerNodeCount = ActiveReadableWorkerNodeCount();
```

Comment:
Is `GetWorkerNodeCount ` is still used anywhere? Let's remove it with a separate PR

Reply:
Will do

Reply:
completed in https://github.com/citusdata/citus/pull/3595